lib/aggregate-translations.js

```diff
@@ -84,10 +86,16 @@ const aggregatedTranslations = (options) => {
   fileSystem.mkdirpSync(outputDirectory);
 
   // Write aggregated translation messages to a file for each locale
-  writeAggregatedTranslations(aggregatedMessages, locales, fileSystem, outputDirectory);
-
-  // Write intl and translations loaders for the specified locales
-  writeI18nLoaders(locales, fileSystem, outputDirectory, format);
+  writeAggregatedTranslations(aggregatedMessages, locales, fileSystem, outputDirectory, !intlShape);
+
+  /**
+   * Detecting react-intl version used. `react-intl` v3 and greater does not export the intlShape.
+   * In order to use these versions of `react-intl` consumers should utilize terra-application v2, which provides these loaders.
+   */
+  if (intlShape !== undefined) {
```

Comment:
What you have absolutely works but I'd probably go a touch more verbose make a named variable `isReactIntl2` to check here. But it works either way 🤷‍♂️

Reply:
Done 😊